nonprofit-board-management.php

```diff
@@ -946,6 +951,39 @@ private function validate_gravatar( $user_email ) {
 
       return $has_valid_avatar;
     }
+
+    /**
+     * Check if the Block Editor is being used.
+     * 
+     * @return bool True if the Block Editor is being used, false if not.
+     */
+    private function is_block_editor_in_use() {
+
+      $use_block_editor = get_option( 'classic-editor-replace' ) === 'no-replace';
+
+      // Check if the Gutenberg plugin is installed and activated.
+      $gutenberg_plugin = ! ( false === has_filter( 'replace_editor', 'gutenberg_init' ) );
+
+      // Block editor is used by default since WordPress 5.0.
+      $block_editor = version_compare( $GLOBALS['wp_version'], '5.0-beta', '>' );
```

Comment:
Do you think we should get rid of the "beta" part of this since at this point 5.0 will be released tomorrow?

Reply:
Sure, I can update this to just be 5.0.

Reply:
This has been updated.